index.html

```diff
@@ -6,22 +6,41 @@
     <link rel="stylesheet" type="text/css" href="src/style.css" />
     <script src="https://code.angularjs.org/tools/system.js"></script>
 
-    <script src="https://code.angularjs.org/2.0.0-beta.0/angular2-polyfills.min.js"></script>
-    <script src="https://code.angularjs.org/2.0.0-beta.0/Rx.min.js"></script>
-    <script src="https://code.angularjs.org/2.0.0-beta.0/angular2.dev.js"></script>
+    <script src="node_modules/reflect-metadata/Reflect.js"></script>
+    <script src="node_modules/zone.js/dist/zone.min.js"></script>
+    <script src="node_modules/rxjs/bundles/Rx.umd.js"></script>
+    <script src="node_modules/@angular/core/core.umd.js"></script>
+    <script src="node_modules/@angular/common/common.umd.js"></script>
+    <script src="node_modules/@angular/compiler/compiler.umd.js"></script>
+    <script src="node_modules/@angular/http/http.umd.js"></script>
+    <script src="node_modules/@angular/platform-browser/platform-browser.umd.js"></script>
+    <script src="node_modules/@angular/platform-browser-dynamic/platform-browser-dynamic.umd.js"></script>
```

Comment:
I'm really thinking we might be better off moving this proj to jspm fully such that we can avoid all these scripts. WDYT? (That's not for this PR, this is great for now!)